package.yaml

```diff
@@ -54,7 +54,7 @@ library:
     - Test.DocTest.Internal.Cabal
 
   dependencies: &dependencies
-    ghc: ">= 8.0 && < 9.12"
+    ghc: ">= 8.0 && < 9.13"
```

Comment:
This was actually correct before. It should be `< 9.14` so that Doctest can be used with development versions of GHC.

Reply:
I'm gonna take care of it.